src/flags/run_time_configurable.cpp

```diff
@@ -54,6 +54,9 @@ DEFINE_double(query_execution_timeout_sec, 600,
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 DEFINE_bool(cartesian_product_enabled, true, "Enable cartesian product expansion.");
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_bool(text_search_enabled, true, "Enable text search.");
```

Comment:
Should be `false` by default (even the "build" default should be `false`)

Reply:
Done (by switching to experimental flags introduced in #1702)